app/uk/gov/hmrc/tai/util/DateTimeHelper.scala

```diff
@@ -27,4 +27,6 @@ object DateTimeHelper {
     dateTimeFormatter.parseLocalDate(date)
   }
 
+
+  implicit val dateTimeOrdering: Ordering[LocalDate] = Ordering.fromLessThan(_ isAfter _)
```

Comment:
So I assume this won't break any other current implicit ordering as they will take precedence over this one

Reply:
Yes, for this to be used it would need to be explicitly be imported